contentreview-impl/tool/pom.xml

```diff
@@ -41,6 +41,10 @@
             <groupId>org.sakaiproject.contentreview</groupId>
             <artifactId>contentreview-service-api</artifactId>
         </dependency>
+		<dependency>
```

Comment:
It's considered bad practice to include the `impl` in the tool as you should be going through the API. If there are utility classes they should not be in the `impl`